docs/custom-metrics-apiservice.yaml

```diff
@@ -1,13 +1,13 @@
-apiVersion: apiregistration.k8s.io/v1beta1
+apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1beta1.custom.metrics.k8s.io
+  name: v1.custom.metrics.k8s.io
 spec:
   service:
     name: kube-metrics-adapter
     namespace: kube-system
   group: custom.metrics.k8s.io
-  version: v1beta1
```

Comment:
same [here](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/kube-metrics-adapter/custom-metrics-apiservice.yaml#L10)

Reply:
This one theoretically defines the version the server talks. Since there's no notable change and we don't strictly define it, I think it's ok too. I'd be interested in updating it on our environment too.

Reply:
same as [above](https://github.com/zalando-incubator/kube-metrics-adapter/pull/310#discussion_r620968482).